lib/em-hiredis/redis_client.rb

```diff
@@ -26,7 +26,8 @@ def initialize(
         uri,
         inactivity_trigger_secs = nil,
         inactivity_response_timeout = nil,
-        em = EventMachine)
+        em = EventMachine,
+        options = {})
```

Comment:
I think `max_reconnect_attempts` would be a better name for this parameter, but I don't feel too strongly.

Reply:
Forget this. https://github.com/redis/redis-rb uses `reconnect_attempts`. Let's be consistent.